docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Hmm, I think there is some overlap between this comment and the comments below.

The best thing in the short term is probably to:

- keep calling all of these things addons, but not mandating an installation method.  Each addon should document its own installation procedure.  This keeps the docs leaner and easier to maintain, since the documentation for each addon _lives with that addon_, and some addons will be more / less complex than others.

- We can link to each of the existing `/cluster/addons/X` addons in this doc.

- As the old `/cluster` directory is torn down, those existing addons will either find new homes or go away.

WDPT?